clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -474,14 +474,14 @@ class StandardGceAttributeValidator {
           utilization.with {
             validateNotEmpty(metric, "${path}.metric")
 
-            if (utilizationTarget <= 0) {
+            if (utilizationTarget!=null && utilizationTarget <= 0) {
               errors.rejectValue("${context}.${path}.utilizationTarget",
                 "${context}.${path}.utilizationTarget must be greater than zero.")
-            }
 
-            validateNotEmpty(utilizationTargetType, "${path}.utilizationTargetType")
+              validateNotEmpty(utilizationTargetType, "${path}.utilizationTargetType")
```

Comment:
You moved the validate not empty under the if check... I'm not sure if that was intentional?  That'd change I think the behavior on this.... Example:
* If utilization target is 5... the validate not empty check would no longer fire.
* That MAY be... irrelevant, but maybe not, not as familiar with the impact on this...